client/packages/common/src/styles/theme.ts

```diff
@@ -172,7 +172,7 @@ export const themeOptions = {
     outline: { main: '#555770' },
     divider: '#eaeaea',
     drawerDivider: '#eaeaea',
-    error: { main: '#e63535' },
+    error: { main: '#e63535', background: '#ffcdce' },
```

Comment:
Needs a lighter colour for the background... tried to find the closest to the screenshot I can....